### DISCUSSION TITLE
Welcome to GitHub Sponsors product feedback!

### DISCUSSION THREAD

**Discussion by bdresser** [community/community#3711] [Other Feature Feedback, Questions, & Ideas]
### Welcome! 👋 
This category is a place for our community to discuss GitHub Sponsors. Have a feature request, a suggestion for improvement, or just want to pass along your "thanks" to the Sponsors team? You're in the right place.

We also welcome you to share your own journey and best practices as a sponsored developer — what's worked for you? How do you grow your sponsorships? What are you building for your community? 

### **Helpful links**
* Check out the [repository README](https://github.com/github/feedback#github-public-feedback-discussions) and our [code of conduct](https://github.com/github/feedback/blob/main/CODE_OF_CONDUCT.md) to learn more about what to expect and how to engage. 
* For bug reports or other user-specific issues, head to https://support.github.com. 
* And for questions on how to set up Sponsors, please head to the [Sponsors product documentation](https://docs.github.com/en/github/supporting-the-open-source-community-with-github-sponsors/about-github-sponsors)

**Comment by oscard0m** [comment]
Thanks for this new release. I think this one time sponsorship will open the door to get some developers in the community some visibility and recognition. 🍾 

---

On the other hand, I'm willing to read what practices and tips sponsored developers can share with the community. I would like to explore this world.

**Comment by aleksey-hoffman** [comment]
Are you guys considering adding Paypal or any other payment processor to increase the "Github Sponsors" availability in other countries in the **near future**?

**Reply by bdresser** [reply]
Hey @aleksey-hoffman – we're considering other payment processors, but don't have any near-term plans to make this happen. Thank you for your patience 🙏

**Reply by kolumnin** [reply]
🙏

**Comment by nuoomnoy02** [comment]
Sup

**Reply by bdresser** [reply]
Sup :)

**Comment by ib568** [comment]
We moved

**Reply by ib568** [reply]
Go

**Comment by sopenefendi** [comment]
Thankyou

**Comment by ali13691369** [comment]
Hi

**Comment by ali13691369** [comment]
> Hi

**Comment by yulimfish** [comment]
Hopefully it will allow us to add pictures, in China we often use WeChat payment codes for appreciation

**Comment by Christopher-Hayes** [comment]
Request - As someone who would sponsor one project I used a lot each month, some sort of rotating sponsorship would be nice. With the current system I start sponsoring someone, then cancel the auto-renew before the month is over, then start a new sponsor subscription with a different project.
I would prefer to just have a dollar amount set, like $10, that GitHub takes out at the start of each month. And there would be like a selection for what project(s) that would go towards, that I can update without having to cancel/restart all this subscription non-sense.

**Reply by Christopher-Hayes** [reply]
I did this for several months. But, the billing was kind of a nuisance. I've recently switched to using https://henrygives.coffee instead which allows me to just have Patreon take out $10 and that goes towards quality open-source projects. But, I would prefer a way I could spread the love across the many packages and projects I currently use.

**Comment by hcharnell7** [comment]
I really need help trying to develop my cyber security software/business. I am very interested in giving major feedback and getting the help on trying to develop my business. Thanks 😊

**Comment by ib568** [comment]
Thanks
